docs/components/cloud/cloudhub.md

```diff
@@ -62,4 +62,4 @@ The CloudHub can be configured in three ways as mentioned below :
 
 - **Start the websocket server only**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-websocket-server-only) to see the details.
 - **Start the quic server only**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-quic-server-only) to see the details.
-- **Start the websocket and quic server at the same time**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-quic-server-only) to see the details
\ No newline at end of file
+- **Start the websocket and quic server at the same time**: Click [here](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md#start-the-websocket-and-quic-server-at-the-same-time) to see the details
```

Comment:
use relative path here.

Reply:
ok

Reply:
BTW there are other places having this issue, e.g. [README.md](https://github.com/kubeedge/kubeedge#in-the-cloud)

Reply:
We can fixe them in a series of follow-up prs :)